.ci/packaging.groovy

```diff
@@ -243,6 +243,20 @@ def publishPackages(baseDir){
   )
 }
 
+/**
+* There is a specific folder structure in https://staging.elastic.co/ and https://artifacts.elastic.co/downloads/
+* therefore the storage bucket in GCP should follow the same folder structure.
+* This is required by https://github.com/elastic/beats-tester
```

Comment:
I guess this is because `filebeat` and `x-pack/filebeat` forlders

```suggestion
* This is required by https://github.com/elastic/beats-tester
* e.g.
* baseDir=name -> return name
* baseDir=name1/name2/name3-> return name2
```

Reply:
indeed, so far there are up to two folders and the nested folder in this particular case should be name

So let's see what the beats team think about this particular PR.